pkg/reconciler/apiserversource/apiserversource.go

```diff
@@ -87,6 +91,9 @@ type Reconciler struct {
 
 	source         string
 	sinkReconciler *duck.SinkReconciler
+	context        context.Context
```

Comment:
I don't think I've ever seen context saved this way. Can we rename this loggingContext or something similar?

Reply:
Right, renamed to `loggingContext`